pyvista/themes.py

```diff
@@ -0,0 +1,1176 @@
+        self._axes['x_color'] = 'tomato'
+        self._axes['y_color'] = 'seagreen'
+        self._axes['z_color'] = 'blue'
+
+
+class ParaViewTheme(DefaultTheme):
+    """Set the theme to a paraview-like theme.
+
+    Examples
+    --------
+    >>> import pyvista
+    >>> from pyvista import themes
+    >>> pyvista.set_plot_theme(themes.ParaViewTheme())
+
+    Alternatively, set via a string.
+
+    >>> pyvista.set_plot_theme('paraview')
+
+    """
+
+    def __init__(self):
+        """Initialize theme."""
+        super().__init__()
+        self._name = 'paraview'
+        self._background = PARAVIEW_BACKGROUND
+        self._cmap = 'coolwarm'
+        self._font['family'] = 'arial'
+        self._font['label_size'] = 16
+        self._font['color'] = 'white'
+        self._show_edges = False
+        self._color = 'white'
+        self._outline_color = 'white'
+        self._edge_color = 'black'
+        self._axes['x_color'] = 'tomato'
+        self._axes['y_color'] = 'gold'
+        self._axes['z_color'] = 'green'
+
+
+class DocumentTheme(DefaultTheme):
+    """Set the global theme to the document theme.
+
+    This theme uses a white background, the "viridis" colormap,
+    disables edges and black fonts.  Best used for presentations,
+    papers, etc.
+
+    Examples
+    --------
+    >>> import pyvista
+    >>> from pyvista import themes
+    >>> pyvista.set_plot_theme(themes.DocumentTheme())
+
+    Alternatively, set via a string.
+
+    >>> pyvista.set_plot_theme('document')
+
+    """
+
+    def __init__(self):
+        """Initialize the theme."""
+        super().__init__()
+        self._name = 'document'
+        self._background = 'white'
+        self._cmap = 'viridis'
+        self._font['size'] = 18
+        self._font['title_size'] = 18
+        self._font['label_size'] = 18
+        self._font['color'] = 'black'
+        self._show_edges = False
+        self._color = 'tan'
+        self._outline_color = 'black'
+        self._edge_color = 'black'
+        self._axes['x_color'] = 'tomato'
+        self._axes['y_color'] = 'seagreen'
+        self._axes['z_color'] = 'blue'
+
+
+class _TestingTheme(DefaultTheme):
+    """Low resolution testing theme for ``pytest``.
+
+    Necessary for image regression.  Xvfb doesn't support
+    multi-sampling, so we disable it here for consistency between
+    desktops and remote testing.
+    """
+
+    def __init__(self):
+        super().__init__()
+        self._name = 'testing'
+        self._multi_samples = 1
+        self._window_size = [400, 400]
+
+
+class _GlobalTheme(DefaultTheme):
+    """Global PyVista theme.
+
+    Examples
+    --------
+    Change the default background color to white.
+
+    >>> import pyvista
+    >>> pyvista.global_theme.color = 'white'
+
+    Show edges by default.
+
+    >>> pyvista.global_theme.show_edges = True
+
+    """
+
+    def __init__(self):
+        """Initialize the base theme."""
+        super().__init__()
```

Comment:
Do we really need a trivial subclass for this? I presume this is for the custom docstring. Perhaps we can just instantiate a `DefaultTheme` and change its `__doc__`? If the user rebinds the global name to a new theme it's going to lose its `_GlobalTheme`ness anyway.

If we do keep the subclass I'm pretty sure we can just stop the class definition after the docstring:
```suggestion
class _GlobalTheme(DefaultTheme):
    """Global PyVista theme.

    Examples
    --------
    Change the default background color to white.

    >>> import pyvista
    >>> pyvista.global_theme.color = 'white'

    Show edges by default.

    >>> pyvista.global_theme.show_edges = True

    """
```

Reply:
Since we're now calling it `global_theme`, I think it's clear what we're using now.  I'll just remove this trival class, especially since it can be overridden or overwritten.